config/webpackDevServer.config.js

```diff
@@ -100,27 +94,5 @@ module.exports = function (proxy, allowedHost) {
     },
     // `proxy` is run between `before` and `after` `webpack-dev-server` hooks
     proxy,
-    onBeforeSetupMiddleware(devServer) {
```

Comment:
My understanding is that these options were removed. Is it safe to simply remove all the logic that was there?

Reply:
- safe - for sure, because the worst thing that could happen is that the dev server doesn't work.
- we don't use proxy
- the error overlay seems to be working fine
- production cache thing is not related to our use case for sure